scripts/newtab.js

```diff
@@ -0,0 +1,188 @@
+'use strict'
+
+var placeholders = [];
+var url = 'https://bwinton.github.io/whimsy/thumbnail-gifs.txt';
+var getting = browser.storage.sync.get('newtab');
+var listener = false;
+var tippyTopSites = require('tippy-top-sites');
+
+initialize();
+
+function onError(error) {
+  console.log(error);
+}
+function loadTopSites(){
+  //display top sites
+  browser.topSites.get().then((topSitesArray) => {
+    var grid = document.querySelector("#newtab-grid");
+    grid.innerHTML = "";
+    for (let topSite of topSitesArray) {
```

Comment:
So, on first run, I only see seven or eight top sites, and they're centered which looks kind of odd…  Could we maybe use [a grid layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout) to make things line up better?  (I think `grid-auto-columns` and `grid-auto-rows` will help with this…)